### PR TITLE
Fix/add featured widget

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Changes

- fix: add featured widget

closes https://github.com/WalletConnect/web3modal/issues/2383

### CODE REVIEW

File: packages/scaffold/src/views/w3m-connect-view/index.ts [hunk 1]
```diff
@@ -49,8 +49,9 @@ export class W3mConnectView extends LitElement {
             <w3m-connect-recent-widget></w3m-connect-recent-widget>
             <w3m-connect-announced-widget></w3m-connect-announced-widget>
             <w3m-connect-injected-widget></w3m-connect-injected-widget>
-            <w3m-connect-coinbase-widget></w3m-connect-coinbase-widget>
+            <w3m-connect-featured-widget></w3m-connect-featured-widget>
```
Comment (enesozturk) [inline packages/scaffold/src/views/w3m-connect-view/index.ts:52]: @svenvoskamp What about the `w3m-connect-coinbase-widget` one? Does the featured widgets including coinbase as well? If yes, should we remove `w3m-connect-coinbase-widget`?

Comment (svenvoskamp) [reply packages/scaffold/src/views/w3m-connect-view/index.ts:52]: @tomiir @enesozturk I think coinbase is part of `w3m-connect-external-widget`? Please correct me if I'm wrong. 

I think we can remove the `w3m-connect-coinbase-widget` and replace it for the external one

Comment (tomiir) [reply packages/scaffold/src/views/w3m-connect-view/index.ts:52]: I think the only external one we have is coinbase. We can remove either of them I think, but CB widget has some logic to hide the duplicated connectors

Comment (svenvoskamp) [reply packages/scaffold/src/views/w3m-connect-view/index.ts:52]: Okay, then I will remove the external one

**Review by enesozturk — APPROVED** [review]

**Pull request closed without merging**